go.sum

```diff
@@ -470,6 +470,8 @@ github.com/tmc/grpc-websocket-proxy v0.0.0-20190109142713-0ad062ec5ee5/go.mod h1
 github.com/tmc/grpc-websocket-proxy v0.0.0-20201229170055-e5319fda7802/go.mod h1:ncp9v5uamzpCO7NfCPTXjqaC+bZgJeR0sMTm6dMHP7U=
 github.com/wind-river/gophercloud v0.0.0-20220503140232-192cf245ebfd h1:SqpOmGVa4q5yHaILQKtLJ1W2/uZuL6Rt3moDK4gXodo=
```

Comment:
I think we may need to remove the reference of the previous gophercloud commit here: v0.0.0-20220503140232-192cf245ebfd, can you remove that and try to make the image?

Reply:
It works, I updated the commit, thanks for the review